src/asset-picker/AssetPickerFilter.js

```diff
@@ -19,10 +20,10 @@ export default class AssetPickerFilter extends Component {
 		actions.updateAssetPickerFilter(e);
 	}
 
-	// componentDidMount() {
-	// 	const assetSearchNode = findDOMNode(this.refs.assetSearch);
-    //     setTimeout(() => assetSearchNode.focus(), 1000);
-    // }
+    componentDidMount() {
+		const assetSearchNode = findDOMNode(this.refs.asset);
+        setTimeout(() => assetSearchNode.firstChild.focus(), 300);
```

Comment:
setTimeout is a hack ~ I was not able to find which part of the app got the focus, seems to be <body>, but I dunno why

Any idea?